themes/finna2/templates/_ui/components/finna-dateinput.phtml

```diff
@@ -0,0 +1,37 @@
+<?php
+  $dateinputId = $this->dateinputId ?? '';
+  $dateinputName = $this->dateinputName ?? '';
+  $dateinputClass = $this->dateinputId ?? '';
```

Comment:
Onko tämä tarkoituksella niin, että id:stä tulee myös luokka eikä muuta voi?

Reply:
Ei ole tarkoituksella. Pitäisi olla $dateinputClass = $this->dateinputClass ?? '';